themes/mlh.zsh-theme

```diff
@@ -0,0 +1,59 @@
+# The Official Theme of 
+##     ## ##       ##     ## 
+###   ### ##       ##     ## 
+#### #### ##       ##     ## 
+## ### ## ##       ######### 
+##     ## ##       ##     ## 
+##     ## ##       ##     ## 
+##     ## ######## ##     ## 
+
+# # # # # # # # # # # # # # # # # #
+# # # Feel free to customize! # # #
+# # # # # # # # # # # # # # # # # #
+
+# To easily discover colors and their codes, type `spectrum_ls` in the terminal
+
+export AT_SYMBOL=" @ "
+export IN_SYMBOL=" in "
+export ON_SYMBOL=" on "
+export SYMBOL="$"
+
+export USER_COLOR="%{$FG[001]%}"
+export DEVICE_COLOR="%{$FG[033]%}"
+export DIR_COLOR="%{$FG[220]%}"
+export BRANCH_COLOR="%{$FG[001]%}"
+export TIME_COLOR="%{$FG[033]%}"
```

Comment:
These don't need `export` since they're used in the same zsh process.

Reply:
Fixed